specs/pipelines/run.spec

```diff
@@ -138,3 +138,40 @@ Steps:
       |S.NO|pipeline_run_name                        |status     |check_lable_propagation|
       |----|-----------------------------------------|-----------|-----------------------|
       |1   |pipelinerun-with-pipelinespec-taskspec-vb|successfull|no                     |
+
+## S2I nodejs pipelinerun: PIPELINES-03-TC08
+Tags: e2e, integration, pipelines, non-admin, s2i
```

Comment:
I would put this test case in `specs/catalog` or `specs/clustertasks`. Then it would become a nice starting point for migration of catalog tests from https://github.com/ppitonak/cluster-tasks-tests to this repo

Reply:
Actually, we would need to update our pipeline runs...let's leave it like this for now but I would definitely move it to a separate directory later.

Reply:
I moved it to separate directory and created a PR for plumbing as well